test/connection/test_connection.py

```diff
@@ -147,6 +147,23 @@ def test__init__(self, mock_requests, mock_refresh_authentication):
             )
             mock_refresh_authentication.assert_called()
 
+        # 200 status_code return and token provided
+        with patch("weaviate.connect.connection.Connection._log_in") as mock_login:
+            connection = Connection(
+                url='test_url',
+                auth_client_secret=None,
+                timeout_config=(2, 20),
+                proxies=None,
+                trust_env=False,
+                additional_headers={'AuthorizatioN': 'my token'},
```

Comment:
is the N at the end of AuthorizatioN a typo?

Reply:
No, header key should be case insensitive.